generators/gae/index.js

```diff
@@ -773,7 +773,7 @@ module.exports = class extends BaseGenerator {
 
                 if (this.buildTool === 'maven') {
                     this.log(chalk.bold('\nRun App Engine DevServer Locally: ./mvnw package appengine:run -DskipTests'));
-                    this.log(chalk.bold('Deploy to App Engine: ./mvnw package appengine:deploy -DskipTests -Pgae,prod-gae'));
+                    this.log(chalk.bold('Deploy to App Engine: ./mvnw package appengine:deploy -DskipTests -Pgae,prod,prod-gae'));
                 } else if (this.buildTool === 'gradle') {
                     this.log(chalk.bold('\nRun App Engine DevServer Locally: ./gradlew appengineRun'));
                     this.log(chalk.bold('Deploy to App Engine: ./gradlew appengineDeploy -Pgae -Pprod-gae'));
```

Comment:
Maybe add -Pprod for Gradle as well?

Reply:
@ludoch : I thought about that just for consistency sake, but it's not needed for gradle as the change (https://github.com/jhipster/generator-jhipster/pull/11207) only effect maven. Do you think we should add it just for make the two commands consistent? :smile: